packages/calcite-components/src/components/modal/modal.e2e.ts

```diff
@@ -123,14 +123,40 @@ describe("calcite-modal properties", () => {
         (elm.beforeClose = (window as typeof window & Pick<typeof elm, "beforeClose">).beforeClose)
     );
     await page.waitForChanges();
-    await modal.setProperty("open", true);
```

Comment:
is there any way to fix this without changing the property name? A prop name change is technically a breaking change.

Reply:
This isn't a breaking change, no prop is being removed.